lib/browser/cookie.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var _ = require('lodash');
+var assert = require('assertive');
+
+var getTestiumCookie = require('testium-cookie').getTestiumCookie;
+
+exports._forwarded = [
+  // TODO: Port validateCookie to webdriver-http-sync
```

Comment:
Yeah, I was trying to keep this PR "small". The TODOs are there to track that this technically contains regressions (parameter validations that were dropped).